src/main/java/de/tum/cit/aet/application/service/ApplicationService.java

```diff
@@ -0,0 +1,124 @@
+package de.tum.cit.aet.application.service;
+
+import de.tum.cit.aet.application.constants.ApplicationState;
+import de.tum.cit.aet.application.domain.Application;
+import de.tum.cit.aet.application.domain.dto.ApplicationForApplicantDTO;
+import de.tum.cit.aet.application.domain.dto.CreateApplicationDTO;
+import de.tum.cit.aet.application.domain.dto.UpdateApplicationDTO;
+import de.tum.cit.aet.application.repository.ApplicationRepository;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class ApplicationService {
+
+    private final ApplicationRepository repository;
+
+    public ApplicationService(ApplicationRepository repository) {
+        this.repository = repository;
+    }
+
+    /**
+     *
+     * @param payload
+     * @return created ApplicationForApplicantDTO
+     */
+    @Transactional
+    public ApplicationForApplicantDTO createApplication(CreateApplicationDTO payload) {
```

Comment:
```suggestion
    public ApplicationForApplicantDTO createApplication(CreateApplicationDTO createApplicationDTO) {
```
Could you please rename the parameter of the createApplication() method? Since it is no longer a Payload record